full-service/src/db/txo.rs

```diff
@@ -354,85 +351,45 @@ impl TxoModel for Txo {
         Ok(txo_id.to_string())
     }
 
-    fn create_minted(
-        output: &TxOut,
-        tx_proposal: &TxProposal,
```

Comment:
Does this still work with the offline flow, where we are copying TxProposals from the offline machine? Or is the assumption that the online machine is never storing data about which TXOs they minted, and that path is now removed?

Reply:
Information about minted txos is still available, but in the TransactionOutputs table. The minted account field in the txos table was a duplicate data entry